src/seasidecache.cpp

```diff
@@ -1420,6 +1438,7 @@ void SeasideCache::startRequest(bool *idleProcessing)
             m_fetchRequest.setFilter(favoriteFilter());
             m_fetchRequest.setFetchHint(favoriteFetchHint(m_fetchTypes));
             m_fetchRequest.setSorting(m_sortOrder);
+            qDebug() << "Starting favorites query at" << m_timer.elapsed() << "ms";
```

Comment:
No argument with these traces, but just so you know: if you export QTCONTACTS_SQLITE_TRACE=1 then the timings of all engine tasks are reported.